R/build-articles.R

```diff
@@ -136,16 +141,30 @@ build_rmarkdown_format <- function(pkg = ".",
     render_page(pkg, "vignette", data, path, depth = depth)
   )
 
-  list(
-    path = path,
-    format = rmarkdown::html_document(
+  format <- rmarkdown::html_document(
       toc = toc,
       toc_depth = 2,
       self_contained = FALSE,
       theme = NULL,
-      template = path
-    )
-  )
+      template = path)
+
+  format$knitr$knit_hooks <-
```

Comment:
Can we extract this bit (and the `colourise_chunk` function) out into a package. Maybe it should live in ansi strings?